File: PHP/Exception/Finally.php
function addRecord($record, $db) {
    try {
      $db->lock($record->table);
      $db->prepare($record->sql);
      $db->exec($record->params);
    } catch(Exception $e) {
        $db->rollback($record);
        if (!write_log($e->getMessage(), $e->getTraceAsString())) {
            throw new Exception('Unable to write to error log.');
        }
    } finally {
        $db->unlock($record->table);
    }
    return true;
}

//https://pt.stackoverflow.com/q/48279/101

File: Java/Exception/Finally.java
public class Main {
	public static void main(String[] args) {
		System.out.println(teste());
	}
	
	public static boolean teste() {
		try {
            System.out.println("Parte 1");
            return false;
        } finally {
            System.out.println("Parte 2");
        }
	}
}

//https://pt.stackoverflow.com/q/42313/101
